### ISSUE TITLE
Tooltips in `StatusChatListAndCategories` are not internationalizable

### ISSUE THREAD

**Issue by PascalPrecht** [status-im/StatusQ#226]
Rendered categories come with some action buttons, one that's supposed to let users add new channels, and another one that opens a context menu for the category.

These buttons come with a tooltip that have a default text respectively and can usually be altered via `StatusChatListCategory` like this:

```
StatusChatListCategory {
  addButton.tooltip.text = qsTr("Add channel inside category")
  menuButton.tooltip.text = qsTr("More")
}
```

At this point however, status chat list categories that are rendered inside a `Repeater` in `StatusChatListAndCategories`, don't expose an API for this.

In other words, when using that component, there's no way to change the tooltip text for the categories from the outisde.
